db/migrate/20160527093031_limit_os_description.rb

```diff
@@ -0,0 +1,12 @@
+class LimitOsDescription < ActiveRecord::Migration
+  def up
+    Operatingsystem.select { |os| os.description.present? }.each do |os|
```

Comment:
Not that it would make much difference since I don't think anyone has a massive amount of OSes in the database, but you can use `Operatingsystem.where("description <> ''")` here